sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -1,11 +1,518 @@
 # Azure Communication Chat Service client library for iOS
-Azure Communication Chat Service
+This package contains a iOS SDK for Azure Communication Services for Chat.
+Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview)
 
-[Source code](TODO: Find source URL) | [API reference documentation](TODO: Find API docs URL) | [Product documentation](TODO: Find product documenation URL) | [Samples](TODO: Find samples URL)
+# Getting started
 
+## Prerequisites
 
+- An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource)
 
-## Contributing
+
+## User and User Access Tokens
+
+User access tokens enable you to build client applications that directly authenticate to Azure Communication Services. Refer [here](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens) to learn how to create a user and issue an User Access Token
+
+The user id `CommunicationUser.identifier`  created above will be used later, because that user should be added as a member of new chat thread when you creating
```

Comment:
nit; Double space.
```suggestion
The user id `CommunicationUser.identifier` created above will be used later, because that user should be added as a member of new chat thread when you creating
```